### PR TITLE
Patient zeros don't roll for natural immunity

### PR DESCRIPTION
**Pull request by Pathid**
In response to https://github.com/vgstation-coders/vgstation13/pull/17295#issuecomment-362259648
:cl:
* tweak: Patients zero of a viral outbreak no longer get a roll for natural immunity to that virus.

### CODE REVIEW

File: code/modules/virus2/disease2.dm [hunk 1]
```diff
@@ -32,7 +33,7 @@ var/global/list/disease2_list = list()
 	e.chance = rand(1, e.max_chance)
 	return e
 
-/datum/disease2/disease/proc/makerandom(var/greater=0)
+/datum/disease2/disease/proc/makerandom(var/greater = FALSE, patient_zero)
```
Comment (MadmanMartian) [inline code/modules/virus2/disease2.dm]: Is it okay to not have the patient zero variable be declared?

Comment (Exxion) [reply code/modules/virus2/disease2.dm]: It doesn't look like it's used

Comment (Pathid) [reply code/modules/virus2/disease2.dm]: I'm doing this in lieu of:
`/datum/disease2/disease/proc/makerandom(var/greater = FALSE, var/pz)`
`patient_zero = pz`
And apparently it works. Or at the very least it doesn't throw errors. If it's bad practice I'll change it.

Comment (Pathid) [reply code/modules/virus2/disease2.dm]: Upon testing it doesn't work.

**Pull request merged**